cfme/test_requirements.py

```diff
@@ -0,0 +1,64 @@
+"""
+    this module contains predefines
+    pytest markers for
+    cfme product requirements
+
+    please import the module instead of elements, i.e.::
+
+        from cfme import test_requirements
+
+        pytestmark = [test_requirements.alert]
+
+        @test_requirments.quota
+        def test_quota_alert():
+            pass
+
+"""
```

Comment:
Something like this will look better.

```python
"""Test requirements mapping

This module contains predefined pytest markers for CFME product requirements.

Please import the module instead of elements:

.. code-block:: python

    from cfme import test_requirements
    
    pytestmark = [test_requirements.alert]
    
    @test_requirements.quota
    def test_quota_alert():
        pass
"""
```